src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2184,4 +2184,7 @@ Do you want to install the latest Dynamo update?</value>
   <data name="RerunButtonToolTip" xml:space="preserve">
     <value>Rerun the graph.</value>
   </data>
+  <data name="MessagePackageMissingDependencies" xml:space="preserve">
```

Comment:
Do we need both resx -es?

Reply:
if you do the standard one it should generate the other I believe.